controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -153,23 +154,16 @@ func (w *Workload) ControlPlaneIsHealthy(ctx context.Context) (HealthCheckResult
 }
 
 // removeMemberForNode removes the etcd member for the node. Removing the etcd
-// member when the cluster has one control plane node is not supported. To allow
-// the removal of a failed etcd member, the etcd API requests are sent to a
-// different node.
+// member when the cluster has one control plane node is not supported.
 func (w *Workload) removeMemberForNode(ctx context.Context, name string) error {
-	// Pick a different node to talk to etcd
 	controlPlaneNodes, err := w.getControlPlaneNodes(ctx)
 	if err != nil {
 		return err
 	}
 	if len(controlPlaneNodes.Items) < 2 {
 		return ErrControlPlaneMinNodes
 	}
-	anotherNode := firstNodeNotMatchingName(name, controlPlaneNodes.Items)
```

Comment:
Can we also remove the related function, or is it used anywhere else?

Reply:
Yup, it was only used by a test that tested it directly